pkg/crd/crd.go

```diff
@@ -42,10 +43,12 @@ func EnsureFissionCRDs(logger *zap.Logger, clientset *apiextensionsclient.Client
 	for _, crdName := range crdsExpected {
 		crd, err := clientset.ApiextensionsV1().CustomResourceDefinitions().Get(context.TODO(), crdName, metav1.GetOptions{})
 		if err != nil {
-			multierror.Append(errs, fmt.Errorf("CRD %s not found: %s", crdName, err))
+			errs := multierror.Append(errs, fmt.Errorf("CRD %s not found: %s", crdName, err))
```

Comment:
This doesn't seems to be a correct fix. I assume the captured error is different than multi error arguments.

We should not exit the program with Fatal.